layerInfo.json

```
@@ -152,14 +141,17 @@
 				]
 			},
 			"ServiceClients": {
-				"dirs": ["packages/service-clients/"],
+				"dirs": ["packages/service-clients/", "@fluidframework/tinylicious-client"],
 				"deps": [
 					"Container-Definitions",
 					"Driver",
 					"Framework",
 					"Runtime",
 					"Routerlicious-Driver",
-					"Server-Shared-Utils"
+					"Server-Shared-Utils",
+					"Loader",
+					"Server-Libs",
+					"Test-Utils"
```

I don't think tinylicious-client actually requires these - can we omit them?

```suggestion
					"Server-Shared-Utils"
```